src/Components/ArticleCard.jsx

```diff
@@ -0,0 +1,40 @@
+import { useState, useEffect } from 'react'
+import axios from 'axios';
+import Header from "./Header";
+import { useParams } from 'react-router-dom';
+
+const ArticleCard = () => {
+    const [article, setArticle] = useState(null);
+    const { article_id } = useParams();
+
+    useEffect(() => {
+        const fetchArticle = () => {
+            axios.get(`https://backend-nc-news-l5zm.onrender.com/api/articles/${article_id}?sort_by=article_id`)
```

Comment:
It would be great if we could extract this axios request to a separate api.js file 👍 
We also shouldn't need to put any sort_by in the request at this point.